include/wx/gdicmn.h

```diff
@@ -469,6 +469,13 @@ class WXDLLIMPEXP_CORE wxRealPoint
 
     wxRealPoint& operator+=(const wxSize& s) { x += s.GetWidth(); y += s.GetHeight(); return *this; }
     wxRealPoint& operator-=(const wxSize& s) { x -= s.GetWidth(); y -= s.GetHeight(); return *this; }
+
+    wxRealPoint& operator+=(double f) { x += f; y += f; return *this; }
+    wxRealPoint& operator-=(double f) { x -= f; y -= f; return *this; }
```

Comment:
Sorry, I disagree with adding these ones, it just doesn't make sense to me to add points and scalars, it's a dimensional error.

Could we please remove those?